linalg/ode.cpp

```diff
@@ -813,9 +813,15 @@ SIAVSolver::Step(Vector &q, Vector &p, double &t, double &dt)
    }
 }
 
-void NewmarkSolver::Init(TimeDependentOperator &_f)
+void SecondOrderODESolver::Init(SecondOrderTimeDependentOperator &f)
```

Comment:
This line generates a compiler error that `SecondOrderODESolver` is not defined.  I couldn't find a definition for that class in the `linalg` directory (at least, not with a simple grep search).  Maybe an updated `linalg/ode.hpp` is missing from this PR?

Reply:
Correct forgot to add file to commit. Fixed now.